From 3f39ee71b0668ac32b8887d181cf53d7b66cfe1b Mon Sep 17 00:00:00 2001
From: Zeeshan Ali Khan <zeenix@gmail.com>
Date: Wed, 3 Sep 2025 18:00:23 +0200
Subject: [PATCH] Hide GNOME sessions

Hide the GNOME sessions.

https://phabricator.endlessm.com/T29401

Based on a patch from Georges Basile Stavracas Neto <georges.stavracas@gmail.com>

* Rebase 51.beta: Upstream has commit 8f0ffced88d1 ("Drop x11 session
                  targets")
  https://github.com/endlessm/eos-build-meta/issues/232
---
 data/gnome.desktop.in.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/gnome.desktop.in.in b/data/gnome.desktop.in.in
index d6c228ae..5f62e95b 100644
--- a/data/gnome.desktop.in.in
+++ b/data/gnome.desktop.in.in
@@ -7,3 +7,4 @@ Type=Application
 DesktopNames=GNOME
 X-GDM-SessionRegisters=true
 X-GDM-CanRunHeadless=true
+Hidden=true
-- 
2.55.0

